metaci/build/flows.py

```diff
@@ -42,3 +46,6 @@ def post_task(self, step, result):
 
         if "robot_outputdir" in result.return_values:
             import_robot_test_results(flowtask, result.return_values["robot_outputdir"])
+
+        if settings.TEST_MANGER_ENABLED:
```

Comment:
I'm thinking this needs to move inside the `if "robot_outputdir"` condition, since for the moment we're only supporting the export of robot results. I think eventually we may want to support apex tests too, but for now, we only want to call this function is if the flag is set and we have robot results.

Reply:
Good point.